react/src/js/components/Consonant/CardsCarousel/CardsCarousel.jsx

```diff
@@ -46,36 +98,35 @@ function CardsCarousel({
     const prev = useRef(null);
     const next = useRef(null);
 
-    let isDown = null;
-    let startX = null;
-    /* eslint-disable-next-line no-unused-vars */
-    let isMouseMove = false;
-    let interactedWith = false;
+    let firstVisibleCard = 1;
```

Comment:
`firstVisibleCard`, `lastVisibleCard` should be hoisted into refs, 

```
const firstVisibleCard = useRef(1);
```

Otherwise these locals get re‑initialized on every render.